src/branchManagement.ts

```diff
@@ -26,8 +40,20 @@ export async function getBranchDetails(): Promise<{ branchNameFinal: string; app
     let branchNameFinal = '';
     //If prefix ends with / then dont add separator
     let finalPrefix = prefix;
-    if(!prefix.endsWith('/')){
-        finalPrefix = `${prefix}${branchNameSeparator}`;
+
+    if(apps.length > 0){
+        if(appFirst){
```

Comment:
might be a good idea to add some comments to explain what happening here. or maybe split it into its own function